src/commands/nun.ts

```diff
@@ -1,4 +1,89 @@
+import process from 'node:process'
+import type { Choice, PromptType } from '@posva/prompts'
+import prompts from '@posva/prompts'
+import { Fzf } from 'fzf'
 import { parseNun } from '../parse'
 import { runCli } from '../runner'
+import { getPackageJSON } from '../fs'
+import { exclude, invariant } from '../utils'
 
-runCli(parseNun)
+import { dump, load } from '../storage'
+
+runCli(async (agent, args, ctx) => {
+  const storage = await load()
+
+  const isLastCmd = args[0] === '-'
+  const isInteractive = !args.length && !ctx?.programmatic
+
+  if (isLastCmd) {
+    if (!storage.lastUninstalledPkg) {
```

Comment:
I am not so sure about the `ni -` feature, as it would be confusing to work across different sessions/projects. The behavior is also unpredictable. I feel it may not be worth have this feature, the interactive mode should cover the need already.

Reply:
Fixed